superset-frontend/src/explore/components/DatasourcePanel/DatasourcePanelDragOption/index.tsx

```diff
@@ -80,7 +80,11 @@ export default function DatasourcePanelDragOption(
   };
 
   return (
-    <DatasourceItemContainer data-test="DatasourcePanelDragOption" ref={drag}>
+    <DatasourceItemContainer
```

Comment:
We can use `styled-component` property instead of inline style. Like this:

```jsx
<DatasourceItemContainer isDragging={isDragging} data-test="DatasourcePanelDragOption" ref={drag} />

const DatasourceItemContainer = styled.div`
  ${({ theme, isDragging }) => css`
    cursor: ${isDragging ? 'grabbing': 'pointer'};
    ...
  `}
`;

```